### PR TITLE
Feature/add course enroll live status

### PR DESCRIPTION
**Pull request by jc-hiroto**

### CODE REVIEW

File: src/containers/CourseResultViewContainer.js [hunk 1]
```diff
@@ -97,7 +123,100 @@ function CourseResultViewContainer() {
   useEffect(() => {
       window.scrollTo(0, 0);
   } ,[]);
+
+  useEffect(() => {
+      async function fetchData() {
+        setIsFetchingCourseStatus(true);
+        const data = await dispatch(getCourseEnrollInfo(isCourseStatusModalOpen));
```
Comment (swh00tw) [inline src/containers/CourseResultViewContainer.js:130]: Should I add try-catch block here in case there are some errors appear when calling API?

Comment (jc-hiroto) [reply src/containers/CourseResultViewContainer.js:130]: Sure, will add it after I finish my meal. 👍

**Review by swh00tw — COMMENTED** [review]
WML...

File: src/containers/CourseResultViewContainer.js (hunk 1 above)
Comment (swh00tw) [reply src/containers/CourseResultViewContainer.js]: Done~

**Pull request merged**